exports/internal_test.go

```diff
@@ -110,7 +110,7 @@ var _ = Context("Set up internal handler", func() {
 			Expect(err).ShouldNot(HaveOccurred())
 
 			exportStatus := exportResponse2["status"].(string)
-			Expect(exportStatus).To(Equal("complete"))
+			Expect(exportStatus).To(Equal("completed_at"))
```

Comment:
This references the status of export being `complete`, which is different from the time it was completed `completed_at`.